### PR TITLE
CVE-2020-1736

### PR DESCRIPTION
**Pull request by pedrohc**

### CODE REVIEW

**Comment by iamleot** [comment]
@pedrohc Is there any upstream references regarding that? Which versions are affected? Is it fixed upstream?

**Comment by pedrohc** [comment]
No, I don't have any other references.

**Comment by iamleot** [comment]
Can you please get in touch with reporters/upstream regarding them?

**Comment by cristina479** [comment]
Red Hat CNA,

Can you provide an affected or fixed version for this CVE ID in order to publish it to the CVE List?

Thank you,
Cristina

**Comment by pedrohc** [comment]
> Red Hat CNA,
> 
> Can you provide an affected or fixed version for this CVE ID in order to publish it to the CVE List?
> 
> Thank you,
> Cristina

Hi, 

Is this satisfactory? This is the same info for the other Ansible PRs which I'll commit after this one is pushed. 

Thank you.

**Comment by iamleot** [comment]
@pedrohc IMHO no, it still pretty cryptic and missing useful details.

File: 2020/1xxx/CVE-2020-1736.json [hunk 1]
```diff
@@ -4,15 +4,68 @@
...
+            }
+        ]
+    },
+    "references": {
+        "reference_data": [
+            {
+                "url": "https://bugzilla.redhat.com/show_bug.cgi?id=CVE-2020-1736",
+                "name": "https://bugzilla.redhat.com/show_bug.cgi?id=CVE-2020-1736",
+                "refsource": "CONFIRM"
+            }
```
Comment (iamleot) [inline 2020/1xxx/CVE-2020-1736.json]: Please add upstream reference:

 <https://github.com/ansible/ansible/issues/67794>

**Comment by iamleot** [comment]
@pedrohc ...adding upstream reference as I suggested in last review should help to track it.

**Comment by pedrohc** [comment]
> @pedrohc ...adding upstream reference as I suggested in last review should help to track it.

I see. Ok, reference added.

**Comment by iamleot** [comment]
Thank you Pedro!

**Pull request merged**